core/components/ui/slideshow/slideshow.tsx

```diff
@@ -1,13 +1,36 @@
 import useEmblaCarousel from 'embla-carousel-react';
 import { ArrowLeft, ArrowRight, Pause, Play } from 'lucide-react';
-import { ComponentPropsWithoutRef, ReactNode, useEffect, useReducer, useState } from 'react';
+import NextImage, { StaticImageData } from 'next/image';
+import { useEffect, useReducer, useState } from 'react';
 
-interface Props extends ComponentPropsWithoutRef<'section'> {
-  slides: ReactNode[];
-  interval?: number;
```

Comment:
Is there a reason why we are dropping this prop too?

Reply:
With this one, I was thinking that it would be nice to be able to configure this from the Makeswift editor.

Reply:
I will add it back.